python/paddle/trainer_config_helpers/layers.py

```diff
@@ -703,8 +703,10 @@ def __enter__(self):
         assert len(self.inputs) == 0
         return self
 
-    def __exit__(self, *args, **kwargs):
-        del args, kwargs  # unused parameter to suppress warning
+    def __exit__(self, exc_type, exc_value, tb):
+        if exc_type != None:
+            traceback.print_exception(exc_type, exc_value, tb)
+            sys.exit(1)
```

Comment:
I changed here because I found that with the original setting, if there is something wrong when defining the mixed layer (e.g., some error in one of the projection layer), then the error message would not reflect the position/reason of the actual error. So I decided to change it so that the relevant error messages will be displayed